app/components/Images/SVGMetadata.tsx

```diff
@@ -1,7 +1,7 @@
 import { useEffect } from "react";
 import { SVG } from '@svgdotjs/svg.js';
 
-export default function SVGMetadata({svg, id}: {svg: any, id: string}) {
```

Comment:
Do you really need `overrideValues` boolean since `area` and `carbonUnits` are optional?